lib/sync.js

```diff
@@ -25,19 +25,24 @@ var defaultIsDir = function isDirectory(dir) {
     return stat.isDirectory();
 };
 
-var maybeUnwrapSymlink = function maybeUnwrapSymlink(x, opts) {
-    if (!opts || !opts.preserveSymlinks) {
-        try {
-            return fs.realpathSync(x);
-        } catch (realPathErr) {
-            if (realPathErr.code !== 'ENOENT') {
-                throw realPathErr;
-            }
+var defaultUnwrapSymlink = function unwrapSymlink(x) {
+    try {
+        return fs.realpathSync(x);
```

Comment:
does this need adjusting now that #217 has landed?

Reply:
yup!